mythril/analysis/modules/integer.py

```diff
@@ -256,45 +282,62 @@ def _handle_return(state: GlobalState) -> None:
             offset, length = get_concrete_int(stack[-1]), get_concrete_int(stack[-2])
         except TypeError:
             return
+
+        """ TODO: Rewrite this
```

Comment:
commenting this might cause false negatives like in
```
function func(uint a, uint b) {
    return a*b;
} 
```
And i think there is 1 swc sample which has this.